tests/test_python3.py

```diff
@@ -63,3 +69,59 @@ def test_python3_with_docker():
     assert output_file.readline() == "[[ 58  64]\n"
     assert output_file.readline() == " [139 154]]"
     output_file.close()
+
+
+@pytest.mark.integration
+def test_python3_streaming():
+    """
+    Tests python3 with output streaming enabled
+    """
+    test_dir = "./temp_test_python3_streaming"
+    os.makedirs(f"{test_dir}", exist_ok=True)
+    test_script_path = f"{test_dir}/test_script.py"
+
+    with open(test_script_path, "w") as test_script_file:
+        script = """
+import datetime
+import time
+print("attempting to print timestamps every second")
+for _ in range(5):
+    message = datetime.datetime.utcnow().isoformat() + "Z"
+    print(message)
+    time.sleep(1)
+with open("./output/output.txt", "w") as f:
+    f.write("Success")
+            """
+        test_script_file.write(script)
```

Comment:
Could this be a normal file instead of a string that's written to a file?

Reply:
It could, so I'll refactor it.